<launch>
    <arg name="landmark_map" default="random"/>
    <arg name="precompute_trajectory" default="true"/>
    <arg name="tight_control" default="false"/>
    
    <!-- Python3 EKF node -->
    <node pkg="ekf_pkg" type="ekf_node.py" name="ekf_node" output="screen" />
    
    <!-- Call main sim driver script. -->
    <include file="$(find data_pkg)/launch/sim_base.launch" >
        <arg name="landmark_map" value="$(arg landmark_map)"/>
        <arg name="precompute_trajectory" value="$(arg precompute_trajectory)"/>
        <arg name="tight_control" value="$(arg tight_control)"/>
    </include>
</launch>